src/node.js

```diff
@@ -388,6 +396,53 @@
       nextTickQueue.push(obj);
       tickInfo[kLength]++;
     }
+
+    // Internal callbacks that must be run when there is no more micro-tasks
+    // scheduled for this turn.
```

Comment:
"this turn" would indicate the entire turn of the event loop, but these are processed in every phase of the event loop.

Man, the naming of `nextTick()` is so unfortunate...